xlators/cluster/dht/src/dht-inode-read.c

```diff
@@ -1661,3 +1665,116 @@ dht_finodelk(call_frame_t *frame, xlator_t *this, const char *volume, fd_t *fd,
 
     return 0;
 }
+
+static int
+dht_seek2(xlator_t *subvol, call_frame_t *frame, int ret)
+{
+    dht_local_t *local = NULL;
+    int op_errno = EINVAL;
+    off_t offset = 0;
+
+    if (!frame || !frame->local)
```

Comment:
Without frame you'll hit
```
gf_msg("stack", GF_LOG_CRITICAL, 0, LG_MSG_FRAME_ERROR, "!frame"); \
```

on STACK_UNWIND_STRICT macro.

So I assume this check is not needed.